src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -138,84 +144,101 @@ public byte[] TransformFinalBlock(byte[] inputBuffer, int inputOffset, int input
             ValidateTransformBlock(inputBuffer, inputOffset, inputCount);
             if (_inputBuffer == null) throw new ObjectDisposedException(null, SR.ObjectDisposed_Generic);
 
-            int effectiveCount;
-            byte[] temp = GetTempBuffer(inputBuffer, inputOffset, inputCount, out effectiveCount);
+            var (temp, tempCount) = GetTempBuffer(inputBuffer, inputOffset, inputCount);
```

Comment:
> Though, I (personally) prefer the single return with an out.

Further `out` can use registers, instead of needing the stack as ValueTuples do, so it may be more performant. (as always: measure if the results are as expected).